src/AnalogInputFirmata.cpp

```diff
@@ -79,7 +80,11 @@ void AnalogInputFirmata::handleCapability(byte pin)
 {
   if (IS_PIN_ANALOG(pin)) {
     Firmata.write(PIN_MODE_ANALOG);
+#ifdef DEFAULT_ADC_RESOLUTION
+    Firmata.write(DEFAULT_ADC_RESOLUTION);
```

Comment:
This should be added for all of the StandardFirmata variants in the [firmata/arduino repo](https://github.com/firmata/arduino) as well.